app/views/quotes/show_quote.html.erb

```diff
@@ -0,0 +1 @@
+<%= render "daily_quote" %>
```

Comment:
Do we want the daily quote to be on a page by itself? Or it to be included in a place where they login or look at their dashboard?

"I want for a sales member to view a "dashboard" of who they talked in the past 24hours, who they need to follow up with, and a daily inspirational quote"